Pod/Classes/ViewControllers/ARComponentViewController.h

```diff
@@ -5,10 +5,19 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface ARComponentViewController : UIViewController
+@protocol ARComponentFullBleedViewSizing
+
+@property (nonatomic, assign) BOOL fullBleed;
+
+@end
+
+@interface ARComponentViewController : UIViewController <ARComponentFullBleedViewSizing>
```

Comment:
I'm not sure that this protocol adds much value - as every component VC supports it, the Eigen subclass can use the ARMenuAware protocol on that side of the fence though

Reply:
This is also used on a non-ARComponentVC, so there's value